depthai_nodes/ml/parsers/utils/yolo.py

```diff
@@ -218,25 +237,47 @@ def parse_yolo_output(
     @type head_id: int
     @param kpts: A single output of keypoints for the given channel.
     @type kpts: np.ndarray
+    @param det_mode: Detection only mode.
+    @type det_mode: bool
+    @param yolo_version: YOLO version.
+    @type yolo_version: Optional[str]
     @return: Parsed output.
     @rtype: np.ndarray
     """
-    na = 1 if anchors is None else len(anchors)  # number of anchors per head
+    na = 1 if anchors is None else len(anchors) // 2  # number of anchors per head
     bs, _, ny, nx = out.shape  # bs - batch size, ny|nx - y and x of grid cells
 
-    grid = make_grid_numpy(ny, nx, na)
+    if yolo_version is not None and yolo_version.lower() == "yolop":
+        grid = make_grid_numpy(ny, nx, 1)
```

Comment:
Can't we also here use `na` instead of specifically 1?

Reply:
No, the `YoloP` is not using `na` value here. It is hardcoded to `1`.
More detailed explanation on [comment](https://github.com/luxonis/depthai-nodes/pull/96#discussion_r1787344543) below.